noitom_hi5_hand_udp_python/Vector4[] eulerAngles
